staging/src/k8s.io/client-go/util/certificate/certificate_store_test.go

```diff
@@ -69,16 +69,16 @@ func TestUpdateSymlinkNewFileNotExist(t *testing.T) {
 		pairNamePrefix: "kubelet",
 	}
 	if err := s.updateSymlink(oldPairFile); err != nil {
-		t.Errorf("Got %v, wanted successful update of the symlink to point to %q", err, oldPairFile)
+		t.Errorf("Got err %v, wanted successful update of the symlink to point to %q", err, oldPairFile)
```

Comment:
Is the "err" needed in the message? If we are adding that, can we change it to "error"?

Reads better that way. :)

Reply:
DONE. thanks